source/macos/AdaptiveCards/AdaptiveCardsTests/Views/ACRViewTests.swift

```diff
@@ -166,18 +206,6 @@ class ACRViewTests: XCTestCase {
         XCTAssertEqual("Value", actionDelegate.dict["Key"] as? String)
     }
     
-    func testInputHandlerWhenisValidFalse() {
-        let testinputHandler = FakeInputHandlingView()
-        testinputHandler.value = "Value"
-        testinputHandler.key = "Key"
-        testinputHandler.isValid = false
-        
-        view.addInputHandler(testinputHandler)
-        view.handleSubmitAction(actionView: NSButton(), dataJson: nil, associatedInputs: true)
-        
-        XCTAssertEqual(0, actionDelegate.dict.count)
-    }
-    
     func testInputHandlerWithMultipleValues() {
```

Comment:
Regression ?? 
Seems to be a valid case with 1.3

Reply:
Yepp
Don't delete this test, I've modified it to make it apt to current situation.
Regression because we have redefined the use of isValid